scripts/status_reporter/status_store.py

```diff
@@ -0,0 +1,137 @@
+import sqlite3
+
+
+class StatusStore:
+    def _chunks(self, data, rows=10000):
```

Comment:
Minor: you could use the `@staticmethod` decorator here.

i.e.

```python
@staticmethod
def _chunks(data, rows=10000):
    for i in range(0, len(data), rows):
        yield data[i : i + rows]
```

Although this has nothing to do with this class; I'd be tempted to pull it out into a helper file.